src/test/kotlin/org/neo4j/graphql/utils/CypherTestSuite.kt

```diff
@@ -20,16 +20,30 @@ class CypherTestSuite(fileName: String) : AsciiDocTestSuite() {
             val line: Int,
             private val request: String,
             private val cypher: String,
+            private val config: QueryContext?,
             private val cypherParams: Map<String, Any?> = emptyMap(),
             private val requestParams: Map<String, Any?> = emptyMap(),
             private val ignore: Boolean) {
 
         fun run(contextProvider: ((requestParams: Map<String, Any?>) -> QueryContext?)?) {
             println(title)
             try {
-                val ctx = contextProvider?.let { it(requestParams) } ?: QueryContext()
+                val ctx = contextProvider?.let { it(requestParams) } ?: config ?: QueryContext()
                 val result = suite.translate(request, requestParams, ctx)
+
```

Comment:
isn't that just noise in the test output? if we need it we should have a flag for enabling these outputs